internal/dashboard/handler/info_handler.go

```diff
@@ -60,16 +60,16 @@ func GetProcessInfo(_ *gin.Context) (*response.DashboardInfo, error) {
 }
 
 // @ID GetStatistics
-// @Summary get telemetry data
-// @Description get telemetry data
+// @Summary get statistic data
+// @Description get statistic data
 // @Tags Info
 // @Accept application/json
 // @Produce application/json
 // @Success 200 object response.APIResponse{data=response.StatisticDataResponse}
 // @Failure 400 object response.APIResponse
 // @Failure 401 object response.APIResponse
 // @Failure 500 object response.APIResponse
-// @Router /api/v1/metrics/telemetry [GET]
+// @Router /api/v1/metrics/statistic [GET]
```

Comment:
why is it still under metrics
/api/v1/statistics ?

Reply:
I thought the statistic data is kind of "metric". Which group do you suggest to put this handler?